sdk/servicebus/service-bus/test/public/atomManagement.spec.ts

```diff
@@ -61,383 +61,522 @@ const newManagementEntity2 = EntityNames.MANAGEMENT_NEW_ENTITY_2;
 type AccessRights = ("Manage" | "Send" | "Listen")[];
 const randomDate = new Date();
 
-/**
- * These tests are just a sanity check that our updates are actually
- * _doing_ something. We've run into some bugs where we've done things like
- * enabled forwarding only to find that forwarding isn't happening even though
- * we can _read_ the attributes back.
- */
-describe("Atom management - forwarding", () => {
-  let serviceBusClient: ServiceBusClientForTests;
-
-  before(() => {
-    serviceBusClient = createServiceBusClientForTests();
-  });
-
-  after(() => serviceBusClient.test.after());
+describe("ATOM APIs", () => {
```

Comment:
To review this file.. Hide whitespace changes

![image](https://user-images.githubusercontent.com/10452642/135358020-dcf08ce3-d4dd-4a46-865c-d40cabfc26bd.png)